base_layer/wallet/src/transaction_service/protocols/transaction_coinbase_monitoring_protocol.rs

```diff
@@ -389,6 +389,7 @@ where TBackend: TransactionBackend + 'static
                             "Coinbase transaction (TxId: {}) not mined yet, still waiting.", self.tx_id,
                         );
                         // Wait out the remainder of the delay before proceeding with next loop
+                        drop(client);
```

Comment:
What is the reason for dropping here?

Reply:
Will add a comment; this is fixing a bug where the RPC connections are not dropped but stay open.